nre/bpf/nodefw/xtask/src/run.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::process::Command;
+
+use anyhow::Context as _;
+use clap::Parser;
+
+use crate::build_ebpf::{build_ebpf, Architecture, Options as BuildOptions};
+
+#[derive(Debug, Parser)]
+pub struct Options {
+    /// Set the endianness of the BPF target
+    #[clap(default_value = "bpfel-unknown-none", long)]
+    pub bpf_target: Architecture,
```

Comment:
Can we not just ask the OS itself? e.g. via [`cfg(target_endian)`](https://doc.rust-lang.org/reference/conditional-compilation.html#target_endian)?

Reply:
Does endianness matter? generally you should be able to write things explicitly ignoring endianness or for network transport things forcing big endian

Reply:
I would prefer to leave it as is. there is some aya specific stuff here.

https://github.com/aya-rs/aya/blob/0f6a7343926b23190483bed49855fdc9bb10988d/xtask/src/codegen/mod.rs#L17-L53